lib/vanagon/platform/dsl.rb

```diff
@@ -198,6 +198,20 @@ def build_hosts(*args)
       # Because single vs plural is annoying to remember
       alias_method :build_host, :build_hosts
 
+      # Set a flag whether this platform is cross-compiled
+      #
+      # @param type [Boolean] true if this platform is cross-compiled
+      def cross_compiled(flag)
```

Comment:
What is the reason behind the parameter passed to this function? if cross_compiled is just meant to set the cross_compiled flag it could have no parameter.